### PR TITLE
Remove semver from resolutions and prevent `npm-force-resolutions` ru…

### PR DESCRIPTION
**Pull request by andrewscfc**
…nning in `npm ci`

No Issue

**Overall change:** 
- Removed semver as it does not appear to be how `npm-force-resolutions` is meant to be used, semver strings being reset in dependency constaints: https://github.com/bbc/psammead/commit/25b67b0d06b8f9dd4885ad7b62047de846bc420d#diff-32607347f8126e6534ebc7ebaec4853d
- Prevented `npm-force-resolutions` running on `npm ci`
  - Please see https://github.com/bbc/simorgh/pull/7840 for full context of why this change is necessary

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

I have ran `npm ci` and `npm install` and experienced consistent results.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -13525,9 +13525,26 @@
       "dev": true,
       "requires": {
         "array-ify": "^1.0.0",
-        "dot-prop": "^4.2.1"
+        "dot-prop": "^5.1.0"
```
Comment (andrewscfc) [inline package-lock.json:13528]: constraints are being reset back to what the packages require

File: package-lock.json [hunk 2]
```diff
@@ -13525,9 +13525,26 @@
...
       },
       "dependencies": {
+        "dot-prop": {
+          "version": "4.2.1",
```
Comment (andrewscfc) [inline package-lock.json:13532]: resolutions are being applied as appropriate by the `npm-force-resolutions`

File: package-lock.json [hunk 3]
```diff
@@ -7759,7 +7759,7 @@
                 "@babel/runtime": "^7.3.1",
                 "highlight.js": "~9.13.0",
                 "lowlight": "~1.11.0",
-                "prismjs": "^1.21.0",
+                "prismjs": "^1.8.4",
```
Comment (karinathomasbbc) [inline package-lock.json:7762]: Is this OK / what we would expect?

**Review by karinathomasbbc — APPROVED** [review]

File: package-lock.json (hunk 3 above)
Comment (andrewscfc) [reply package-lock.json:7762]: yeah, now the we don't run `npx npm-force-resolutions` when we run `npm ci`, the corrections made to the `package-lock.json` are now being retained. `npm-force-resolutions` was previously modifiying the constraints and `npm ci` didn't set it back again. Now `npm ci` doesn't run `npm-force-resolutions` there's nothing to correct. 🤯

Comment (karinathomasbbc) [reply package-lock.json:7762]: 🤯 ⚡

**Review by simonsinclair — APPROVED** [review]

**Pull request merged**